.github/workflows/clarity-js-sdk-pr.yml

```diff
@@ -11,12 +11,13 @@ env:
 on:
   release:
     types:
-      - published
+      - released
   workflow_dispatch:
 
-jobs:      
+jobs:
   run:
     runs-on: ubuntu-latest
+    if: "!contains(github.ref, '-rc')"
```

Comment:
What is this line doing?

Reply:
It's a condition to ensure the newly created release does not have `-rc` in its name, otherwise it'll exit. We don't want the `clarity-native-bin` to use release candidate pre-releases, so this is how it's being circumnavigated for now. This is sort of a hack to make up for maintenance that needs to be completed on the main `stacks-blockchain` workflow.

Ideally the workflow would create a pre-release instead of an actual release when a tag with `-rc` in the name is passed in. If it worked this way, then the type change from `published` to `released` would've sufficed (see my other comment). But since it doesn't do that at the moment, and instead it creates a real release which I then have to modify later to make it a pre-release.

Once the workflow is improved to create pre-releases when appropriate, this condition can be removed.

Reply:
Cool, could possibly be helpful to leave a short comment with some of this info in the yaml file, but up to you!

Reply:
Good idea, comment added!